app/models/health/tasks/import_epic.rb

```diff
@@ -26,29 +25,33 @@ def initialize(logger: Rails.logger, load_locally: false, configs: nil, prevent_
       @to_revoke = []
       @to_restore = []
       @new_patients = []
-      @configs = configs
+      @configs = configs || Health::ImportConfig.all
       @prevent_massive_change = prevent_massive_change
+
+      raise 'load_locally only supports a single config' if @load_locally && @configs.keys.count > 1
     end
 
     def run!
-      @configs ||= YAML::load(ERB.new(File.read(Rails.root.join("config","health_sftp.yml"))).result)[Rails.env]
```

Comment:
Nice! does this mean we can remove this from .env?

Reply:
Yes... after the migration is run to move the contents into the DB :-)